### ISSUE TITLE
a11y view content is missing in greenhouse-effect

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/chipper#1664]
Lots of content in greenhouse-effect is missing from the "Formatted" a11y view tab. I have never seen this before in other sims. The content is present in the "legacy" and "development" views, so it must be something about the formatting.
